packages/rx_bloc_list/lib/src/widgets/rx_paginated_builder.dart

```diff
@@ -232,22 +242,30 @@ class _RxPaginatedBuilderState<B extends RxBlocTypeBase, T>
   Widget build(BuildContext context) {
     final bloc = RxBlocProvider.of<B>(context);
 
-    return NotificationListener<ScrollNotification>(
-      onNotification: (ScrollNotification scrollInfo) =>
-          _onScrollNotification(scrollInfo, bloc),
-      child: RxBlocBuilder<B, PaginatedList<T>>(
-          bloc: bloc,
-          state: widget.state,
-          builder: (context, snapshot, bloc) {
-            final buildChild = _buildChild(context, snapshot, bloc);
-
-            if (widget.wrapperBuilder != null)
-              return widget.wrapperBuilder!.call(context, bloc, buildChild);
-            return buildChild;
-          }),
+    return RxBlocBuilder<B, PaginatedList<T>>(
+      bloc: bloc,
+      state: widget.state,
+      builder: (context, snapshot, bloc) =>
+          NotificationListener<ScrollNotification>(
+        onNotification: (ScrollNotification scrollInfo) =>
+            _onScrollNotification(scrollInfo, bloc, snapshot),
+        child: _buildChildWithWrapper(context, snapshot, bloc),
+      ),
     );
   }
 
+  Widget _buildChildWithWrapper(
+    BuildContext context,
+    AsyncSnapshot<PaginatedList<T>> snapshot,
+    B bloc,
+  ) {
+    final buildChild = _buildChild(context, snapshot, bloc);
+
+    if (widget.wrapperBuilder != null)
+      return widget.wrapperBuilder!.call(context, bloc, buildChild);
```

Comment:
Use curly braces in control flow structures.

```suggestion
    if (widget.wrapperBuilder != null) {
      return widget.wrapperBuilder!.call(context, bloc, buildChild);
    }

```